### PR TITLE
Add homepage and team updates

### PR DESCRIPTION
**Pull request by lhorie**

### CODE REVIEW

**Comment by gergelyke** [comment]
@lhorie can we add this back? https://github.com/fusionjs/fusionjs.github.io/commit/136339257179711841d187161ff43f53fe0d1ea7

File: src/team.js [hunk 1]
```diff
@@ -8,56 +8,81 @@ import lhorie from './images/lhorie.jpg';
...
+    face: olegg,
+    name: 'Oleg Gomozov',
+    username: 'blackswanny',
+  },
+  {
+    face: gergelyn,
+    name: 'Gergely Nemeth',
+    username: 'gergelyke',
+  },
+
```
Comment (derekjuber) [inline src/team.js]: Extra space here

File: documentation/docs/getting-started/environment-setup.md [hunk 2]
```diff
@@ -1,85 +0,0 @@
----
```
Comment (KevinGrandon) [inline documentation/docs/getting-started/environment-setup.md]: What's the reason for deleting this page? Do we not think it's necessary?

Comment (lhorie) [reply documentation/docs/getting-started/environment-setup.md]: I deleted it because it's pretty uber-specific

**Pull request merged**